README.md

```diff
@@ -89,13 +90,15 @@ npx eslint .
 - Twitter: [@twitterhandle](https://twitter.com/tracy_muso)
 - LinkedIn: [@LinkedIn](https://linkedin.com/in/tracy-musongole)
 
-## 🤝 Contributing
 👤 **Festus Enning**
 
-- GitHub: [@githubhandle](https://github.com/elfin-git)
-- Twitter: [@twitterhandle](https://twitter.com/tracy_muso)
-- LinkedIn: [@LinkedIn](https://linkedin.com/in/tracy-musongole)
+- GitHub: [@githubhandle](https://github.com/Enning94)
+- Twitter: [@twitterhandle](https://twitter.com/nana_akyerefi)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/enning-festus)
 
+
+
+## 🤝 Contributing
 Contributions, issues, and feature requests are welcome!
 
 
```

Comment:
### Notice
Kindly make sure for all your next **GROUP CAPSTONES** to correctly use **Gitflow**. The pull request link that you should submit should suppose to be opened from the `dev` branch into the `main` branch and include all the changes you did for the whole project. For more details I suggest you read this **[article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/code_review_flow_group_projects.md)**